.github/workflows/build-emsdk.yml

```diff
@@ -1,27 +1,78 @@
-name: build-emsdk
+name: Build WASM
 
-on: [push]
+# Run CI only on changes to main branch, or any PR to main. Do not run CI on 
+# any other branch. Also, skip any non-source changes from running on CI
+on:
+  push:
+    branches: main
+    paths-ignore:
+      - 'docs/**'
+      - 'examples/**'
+      - '.gitignore'
+      - '*.rst'
+      - '*.md'
+      - '.github/workflows/*.yml'
+      # re-include current file to not be excluded
+      - '!.github/workflows/build-emsdk.yml'
+
+  pull_request:
+    branches: main
+    paths-ignore:
+      - 'docs/**'
+      - 'examples/**'
+      - '.gitignore'
+      - '*.rst'
+      - '*.md'
+      - '.github/workflows/*.yml'
+      # re-include current file to not be excluded
+      - '!.github/workflows/build-emsdk.yml'
 
 jobs:
   build:
     runs-on: ubuntu-latest
+    env:
+      # pin SDK version to the latest, update manually
+      SDK_VERSION: 0.0.1
+      SDK_ARCHIVE: python-wasm-sdk-stable.tar.bz2
+
+      # use the most recent cython from github, but pin on a commit for CI
+      # stability. This is also needed to benefit from caching cython installs
+      LATEST_CYTHON_COMMIT: 6ac2422b48b689b021a48dff9ee14095232baafe
+
     steps:
-    - uses: actions/checkout@v1
-    - name: pygame-wasm-builder prepare
+    - uses: actions/checkout@v3.0.2
+    
+    - name: Cache pip directory for cython
+      uses: actions/cache@v3
+      with:
+        path: ~/.cache/pip
+        key: wasm-ubuntu-cython-${{ env.LATEST_CYTHON_COMMIT }}
+
+    - name: Install latest cython and regen
       run: |
-            WD=$(pwd)
-            python3 -V
-            echo $WD
-            clang --version | head -n 1
-            echo
-            grep "^Pkg.Revision =" ${ANDROID_HOME}/ndk-bundle/source.properties
-            sudo apt-get update
-            sudo apt-get install bash wget
-            cd $GITHUB_WORKSPACE/..
-            git clone https://github.com/pmp-p/python-wasm-plus.git
-            mkdir $GITHUB_WORKSPACE/../python-wasm-plus/src
-            ln -s $(pwd)/pygame $GITHUB_WORKSPACE/../python-wasm-plus/src/pygame-wasm
-            cd python-wasm-plus
-            bash ./python-wasm-plus.sh
-            bash ./buildapp.sh
+        pip install git+https://github.com/cython/cython.git@${{ env.LATEST_CYTHON_COMMIT }}
+        touch $(find |grep pxd$)
+        python3 setup.py cython_only
 
+    - name: Install python-wasm-sdk
+      run: |
+        mkdir python-wasm-sdk
```

Comment:
At that point it's the parent directory of the pygame source directory, so it should be `/home/runner/work/pygame`

Reply:
so there should be mkdir /home/runner/work/python-wasm-sdk && cd  /home/runner/work/python-wasm-sdk before the mkdir no ?

Reply:
Yeah you are right. I initially thought it'd be fine to have the sdk thing in `/home/runner/work/pygame/python-wasm-sdk`... but since paths are hardcoded on the sdk end, I should fix that here